serialize_test.go

```diff
@@ -1,11 +1,60 @@
 package tiledb
 
 import (
+	"bytes"
 	"runtime"
 	"runtime/debug"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
+func TestSerializeArraySchemaGC(t *testing.T) {
+	disableGC(t)
+
+	ctx, err := NewContext(nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	schema, err := NewArraySchema(ctx, TILEDB_DENSE)
+	if err != nil {
+		t.Fatal(err)
+	}
+	dom, err := NewDomain(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	dim, err := NewDimension(ctx, "d1", TILEDB_INT32, []int32{1, 10}, int32(2))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := dom.AddDimensions(dim); err != nil {
+		t.Fatal(err)
+	}
+	if err := schema.SetDomain(dom); err != nil {
+		t.Fatal(err)
+	}
+	attr, err := NewAttribute(ctx, "a1", TILEDB_INT32)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := schema.AddAttributes(attr); err != nil {
+		t.Fatal(err)
+	}
+	buffer, err := SerializeArraySchemaToBuffer(schema, TILEDB_CAPNP, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	bytes := &bytes.Buffer{}
+	_, err = buffer.WriteTo(bytes)
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.NotEmpty(t, bytes.Bytes())
+	runtime.GC()
+	runtime.GC()
```

Comment:
I don't understand the purpose of the last few lines in this test, but I see there was one like it just below so I'm probably missing something. 

Should there be a check here that `buffer` was garbage collected, or is the test verifying something else with the explicit calls to `runtime.GC()`?

Reply:
I copied the GC calls from the existing test, I don't know what their purpose was.